controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -616,6 +619,9 @@ func (r *SriovNetworkNodePolicyReconciler) renderDevicePluginConfigData(pl *srio
 			if len(p.Spec.NicSelector.PfNames) > 0 {
 				netDeviceSelectors.PfNames = append(netDeviceSelectors.PfNames, p.Spec.NicSelector.PfNames...)
 			}
+			if l := len(p.Spec.NicSelector.LinkTypes); l > 0 {
```

Comment:
why you need the l. can you just write 
"if len(p.Spec.NicSelector.LinkTypes) > 0" like the others?